keel-core/src/main/kotlin/com/netflix/spinnaker/keel/actuation/ResourceActuator.kt

```diff
@@ -200,7 +200,7 @@ class ResourceActuator(
           }
         }
       } catch (e: ResourceCurrentlyUnresolvable) {
-        log.warn("Resource check for {} failed (hopefully temporarily) due to {}", id, e.message, e)
+        log.warn("Resource check for {} failed (hopefully temporarily) due to {}", id, e.message)
```

Comment:
This was causing stack traces to be printed for a case that's an expected condition. It was just noise in the logs.